tensorboard/webapp/runs/views/runs_table/runs_data_table.ng.html

```diff
@@ -42,6 +42,7 @@
         [hparamsEnabled]="true"
         [controlsEnabled]="header.type !== ColumnHeaderType.COLOR && header.type !== ColumnHeaderType.CUSTOM"
         (contextmenu)="openContextMenu($event, header)"
+        [ngClass]="header.type === ColumnHeaderType.COLOR ? 'last-header' : ''"
```

Comment:
I think this could be done using only CSS with a `:last-child` or `:last-of-type` selector